powerauth-webflow-client/src/main/resources/templates/home.html

```diff
@@ -38,134 +38,66 @@
-                    </p>
-                    <form id="disconnect" th:action="@{/connect/demo}" method="post">
-                        <input type="hidden" name="_method" value="delete"/>
-                        <button type="submit" class="btn btn-lg btn-danger">Log out</button>
-                    </form>
-                </div>
+                <table class="operations-table">
+                    <tr>
+                        <td>
+                            <ul class="nav nav-pills nav-stacked" role="tablist">
+                                <li th:each="opConfig : ${availableOperations}" role="presentation" th:classappend="${opConfig.isDefault()} ? 'active' : ''">
+                                    <a th:text="${opConfig.getName()}" th:href="'#'+${opConfig.getType().name()}" th:attr="aria-controls=${opConfig.getType().name()}" role="tab" data-toggle="tab">NAME</a>
+                                </li>
+                            </ul>
+                        </td>
+                        <td>
+                            <div class="tab-content">
+                                <div th:each="opConfig : ${availableOperations}" th:classappend="${opConfig.isDefault()} ? 'active' : ''" role="tabpanel" class="tab-pane" th:id="${opConfig.getType().name()}">
+                                    <div th:if="${opConfig.getType().name() == 'LOGIN'}" class="padding text-left">
 
-                <div th:if="${#lists.isEmpty(connectionMap[__${providerId}__])}">
-                    <div>
-                        <table class="operations-table">
-                            <tr>
-                                <td>
-                                    <ul class="nav nav-pills nav-stacked" role="tablist">
-                                        <li th:each="opConfig : ${availableOperations}" role="presentation" th:classappend="${opConfig.isDefault()} ? 'active' : ''">
-                                            <a th:text="${opConfig.getName()}" th:href="'#'+${opConfig.getType().name()}" th:attr="aria-controls=${opConfig.getType().name()}" role="tab" data-toggle="tab">NAME</a>
-                                        </li>
-                                    </ul>
-                                </td>
-                                <td>
-                                    <div class="tab-content">
-                                        <div th:each="opConfig : ${availableOperations}" th:classappend="${opConfig.isDefault()} ? 'active' : ''" role="tabpanel" class="tab-pane" th:id="${opConfig.getType().name()}">
-                                            <div th:if="${opConfig.getType().name() == 'PAYMENT'}" class="padding">
+                                        <div th:if="${#lists.isEmpty(connectionMap[__${providerId}__])}">
+                                            <p>
+                                                Zvolte typ připojení třetí strany:
```

Comment:
This HTML file contains hardcoded strings in Czech language. Probably not a good idea.